### PR TITLE
Creation of  Todo List App with ES6 Features and Webpack

### PR DESCRIPTION
**Pull request by RaviTejaM9602**
> ES6 Features(Import and Export)
> Event Handling 
> DOM Manipulation
> HTML5 Semantic Tags

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,14 +1,33 @@
...
-   return element;
- }
+const createList = () => {
+  toDoTasks.forEach((task) => {
+    const element = document.createElement('li');
+    element.innerHTML = `<div class="left"><i class="far fa-square"></i> ${task.description} </div>
+    <i class="fa fa-ellipsis-v"></i>`;
+    element.classList.add('rows');
+    List.appendChild(element);
+  });
```
Comment (SajjadAhmad14) [inline src/index.js]: Please make sure that the tasks appear in order of the index values for each task. They are currently being rendered in the order of the Array index. This feature would be helpful later when deleting tasks as they will readjust themselves using the `index` value of the task.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Ravi, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by rustam-tolipov — APPROVED** [review]
## Hi @RaviTejaM9602 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mikethreels — APPROVED** [review]
Hi Ravi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**